cmd/dcrdata/views/ticketpool.tmpl

```diff
@@ -37,7 +37,7 @@
         <br>
         <div class="justify-content-between">
           <div class="dygraph-label dygraph-title mb-1">Distribution of Tickets by Reward Outputs</div>
-          <div class="row col-lg-12 col-sm-8 d-flex text-center m-auto">
+          <div class="col d-flex text-center mw50 w-100 m-auto ">
```

Comment:
`col-lg-12` used to apply a `max-width: 50%` setting, which changed to `width: 50%` with the update. This sets it back to how dcrdata used it with the `mw50` helper